digid_eherkenning/saml2/soap_logout_request.py

```diff
@@ -0,0 +1,169 @@
+from onelogin.saml2.constants import OneLogin_Saml2_Constants
```

Comment:
Would it make more sense to add this to https://github.com/maykinmedia/python3-saml ?

Reply:
Refactored :)